### ISSUE TITLE
arizona.js: allow `subscribe` to "Subscribe to global events."

### ISSUE THREAD

**Issue by paulo-ferraz-oliveira** [arizona-framework/arizona#110]

**Comment by paulo-ferraz-oliveira** [comment]
I'm gonna take this as a starting point to look at the JavaScript interface. It's possible I break a couple of things as we discussed already.

**Comment by paulo-ferraz-oliveira** [comment]
Also

```javascript
arizona.subscribe("reload", () => {
  location.reload();
});
```

should probably be internal to `arizona.js` so we get rid of an extra `.js` 😄

**Comment by williamthome** [comment]
> Also
> 
> ```js
> arizona.subscribe("reload", () => {
>   location.reload();
> });
> ```
> 
> should probably be internal to `arizona.js` so we get rid of an extra `.js` 😄

This piece of code is from [arizona-live-reload.js](https://github.com/arizona-framework/arizona/blob/main/priv/static/assets/js/arizona-live-reload.js), so it's a development thing. It should be optional. We have not defined/discussed an interface for the live-reload, and this file may grow (or not). I don't see it as a problem right now. We could change this later.

**Comment by paulo-ferraz-oliveira** [comment]
Yeah, I saw it being called separately in the code, so I won't touch it for now.
